src/data-source.handlers.ts

```diff
@@ -16,12 +18,15 @@ export const dataSourceHandlers: ResourceHandlerMap = {
     const data = omit(req.body, 'id');
     new DataSourceModel(data)
       .save()
-      .then(datasource => {
+      .then(doc => {
         res
-          .location(datasource.id)
+          .location(doc.id)
           .status(201)
           .send();
+
```

Comment:
hva jeg tenkte, at de to prosesene du gjør er sammenligbar nivå
`
then(doc=>{
   op1(doc);
   op2(doc);
})
`